1657395: Add descriptions for SEO (3 of 3) (#18321)
dotnet/docs: docs/framework/windows-workflow-foundation/using-workflowinvoker-and-workflowapplication.md

---
title: "Using WorkflowInvoker and WorkflowApplication"

> description: This article describes workflow hosting with WorkflowInvoker and WorkflowApplication, two of the methods of hosting that Windows Workflow Foundation provides.

Reviewer: ```suggestion
description: This article describes workflow hosting using WorkflowInvoker and WorkflowApplication.
```

Author: @gewarren - How about this:

This article describes workflow hosting using WorkflowInvoker and WorkflowApplication in Windows Workflow Foundation.

Reviewer: Sounds good.
